pkg/kubelet/cm/dra/plugin/client.go

```diff
@@ -42,13 +52,58 @@ func NewDRAPluginClient(pluginName string) (drapb.NodeClient, error) {
 	return existingPlugin, nil
 }
 
-func (p *plugin) NodePrepareResources(
+type Plugin struct {
```

Comment:
Previously, the implementation of `Plugin` was scattered across `client.go` and `plugin.go`. I moved everything here because I find that more consistent and easier to understand.

/cc @klueska

Reply:
Let's rename this file to plugin.go then. And the other file to registration.go

Reply:
Can we do that in a future, stand-alone PR after all pending changes are merged, including https://github.com/kubernetes/kubernetes/pull/125488? Otherwise it'll be hard to review changes and resolve merge conflicts.

Reply:
Sure.

Reply:
https://github.com/kubernetes/kubernetes/issues/125924